Apps/W1/EDocumentConnectors/FORNAV/app/app.json

```diff
@@ -0,0 +1,51 @@
+{
+	"id": "0addc017-80c4-40ad-bac6-5852f4fc4c55",
+	"name": "Peppol Core",
+	"publisher": "ForNAV",
+	"brief": "",
+	"description": "",
+	"version": "1.0.0.0",
+	"privacyStatement": "",
+	"EULA": "",
+	"help": "",
+	"url": "https://www.fornav.com",
```

Comment:
Look at the descriptions from other apps, take the same url etc. This file has to much deviation. Also version should be 27 and not version 1.

Publisher is Microsoft.
Verify that it compile with BcApps ruleset in settings.json.